pulsar-client-cpp/include/pulsar/ProducerConfiguration.h

```diff
@@ -78,6 +78,18 @@ class ProducerConfiguration {
     ProducerConfiguration& setBatchingMaxPublishDelayMs(
             const unsigned long& batchingMaxPublishDelayMs);
     const unsigned long& getBatchingMaxPublishDelayMs() const;
+
+    /*
+     * Initialize stats interval in seconds. Stats are printed and reset after every 'statsIntervalInSeconds'.
+     * Set to 0 in order to disable stats collection.
+     */
+    ProducerConfiguration& setStatsIntervalInSeconds(const uint64_t&);
```

Comment:
why isn't this a client configuration? i think it's unlikely i'm going to want different intervals for a particular producer and/or consumer in the case where i have several (thousand) of them.